README.md

```diff
@@ -564,29 +564,29 @@ For remote overrides, the following extended settings must be configured:
+  Many deployments of Nightscout use free tier of MongoDB Atlas on Heroku, which is limited in size. After some time, as the volume of stored data grows, it may happen that this limit is reached and system is unable to store new data. This plugin provides pill that indicates size of the Database and shows (when configured) alarms regarding reaching space limit.
 
-  **IMPORTANT:** This plugin can only check how much space database already takes, _but cannot infer_ max size available on server for it. To have correct alarms and realistic percentage, `DBSIZE_MAX` need to be properly set - according to your mongoDB hosting configuration.
+  **IMPORTANT:** This plugin can only check how much space the database already takes, _but cannot infer_ max size available on server for it. To have correct alarms and realistic percentage, `DBSIZE_MAX` need to be properly set - according to your mongoDB hosting configuration.
 
-  **NOTE:** This plugin rely on db.stats() for reporting _logical_ size of database, which may be different than _physical_ size of database on server. It may work for free tier of MongoDB on Atlas, since it calculate quota according to logical size too, but may fail for other hostings or self-hosted database with quota based on physical size. 
+  **NOTE:** This plugin relies on db.stats() for reporting _logical_ size of database, which may be different than _physical_ size of the database on server. It may work for free tier of MongoDB on Atlas, since it calculates quota according to logical size too, but may fail for other hostings or self-hosted database with quota based on physical size. 
   
-  **NOTE:** MongoDB Atlas quota is for **all** databases in cluster, while each instance will get only size of **its own database only**. It is ok when you only have **one** database in cluster (most common scenario) but will not work for multiple parallel databases. In such case, spliting known quota equally beetween databases and setting `DBSIZE_MAX` to that fraction may help, but wont be precise.
+  **NOTE:** MongoDB Atlas quota is for **all** databases in cluster, while each instance will get only size of **its own database only**. It is ok when you only have **one** database in cluster (most common scenario) but will not work for multiple parallel databases. In such case, splitting known quota equally between databases and setting `DBSIZE_MAX` to that fraction may help, but won't be precise.
 
   All sizes are expressed as integers, in _Mebibytes_ `1 MiB == 1024 KiB == 1024*1024 B`
 
-  * `DBSIZE_MAX` (`496`) - Maximal allowed size of database on your mongoDB server, in MiB. You need to adjust that value to match your database hosting limits - default value is for standard Heroku mongoDB free tier.
-  * `DBSIZE_WARN_PERCENTAGE` (`60`) - Threshold to show first warning about database size. When database reach this percentage of `DBSIZE_MAX` size - pill will show size in yellow. 
```

Comment:
I think it makes more sense to change this to "The pill" instead of removing "pill" altogether.